tensorflow/tools/ci_build/builds/ios.sh

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env bash
+# Copyright 2015 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+
+set -e
+
+# Remove any old files first.
+make -f tensorflow/contrib/makefile/Makefile clean
```

Comment:
I think it's probably better to add the file in "tensorflow/contrib/makefile". First, it would make the relative path above less funny. Second, it makes it easier for users interested in iOS builds to find this script.